Backend/database/migrations/0001_01_01_000000_create_users_table.php

```diff
@@ -15,30 +15,15 @@ public function up(): void
             $table->string("email")->unique();
             $table->timestamp("email_verified_at")->nullable();
             $table->string("password");
+            $table->string("role")->default("user");
+            $table->json("favorites")->nullable();
             $table->rememberToken();
             $table->timestamps();
         });
-
-        Schema::create("password_reset_tokens", function (Blueprint $table): void {
-            $table->string("email")->primary();
-            $table->string("token");
-            $table->timestamp("created_at")->nullable();
-        });
-
-        Schema::create("sessions", function (Blueprint $table): void {
-            $table->string("id")->primary();
-            $table->foreignId("user_id")->nullable()->index();
-            $table->string("ip_address", 45)->nullable();
-            $table->text("user_agent")->nullable();
-            $table->longText("payload");
-            $table->integer("last_activity")->index();
-        });
```

Comment:
we don't need information about user session ? I guess it is not that important but it could be useful in monitoring user activity.

Reply:
I honestly don't know, if at a later stage it turns out to be necessary I will restore it

Reply:
For me I think admin will be determined  in a database maybe so he will have immediately access into admin panel and user will register / login only to add a heart to his favourite kebab. But it will be discussed later ✔️